docs/in_production.rst

```diff
@@ -64,4 +67,63 @@ Configuring realtime data ingestion
 Monitoring Pinot
 ~~~~~~~~~~~~~~~~
 
+In order for Pinot to provide effective service there is a core set of metrics which should be monitored to ensure service stability, fault tolerance and acceptable response times. In the section following, there are service level metrics which are recommended to be monitored.
```

Comment:
Are you going to fill in the empty sections above in a future PR?

Reply:
I might word it like: "Pinot exposes several metrics to monitor the service and ensure that pinot users are not experiencing issues. In this section we discuss some of the key metrics that are useful to monitor. A full list of metrics is available in <code classes>"

Reply:
The empty section above are not assigned to me. I just filled in the outline as it was in the repo.

Reply:
Re: monitoring wording

I've removed my 2 paragraphs and replaced with your quoted text.